scripts/00370-sandy.js

```diff
@@ -0,0 +1,5 @@
+let name = "Goodnews Sandy";
+let id = "00370"
```

Comment:
Hi, 

Please modify this to have your full HNG id in the form `HNG-00370'

Reply:
DONE